### DISCUSSION TITLE
Tag Studio for emails

### DISCUSSION THREAD

**Discussion by concimuscb** [TagStudioDev/TagStudio#573] [Ideas]
The exact same problem that TagStudio solves for files, also happens for emails. Anyone that has worked in a position that handles a lot of communication via email knows the pain.

I understand that this is completely out of scope, but it would be amazing if the same idea could be developed as an Outlook extension for example.

**Comment by Nginearing** [comment]
Hi, I might be interested in working on this, can you elaborate further on what it would do exactly?
(As an extension)

**Reply by concimuscb** [reply]
The underlying system is greatly explained here: https://www.youtube.com/watch?v=wTQeMkYRMcw from 1:30 to 6:30.

Essentially it would be an extension that supports a tagging system like the one described in the video + searching functionality to locate emails with those tags.

**Comment by python357-1** [comment]
Unfortunately this does seem like the type of thing that is better suited for an extension, which are likely a long way off. I did open an issue the other day for stuff just like this - where an extension would be managing items in tagstudio that aren't files (#588) - but it isn't even on the roadmap yet, and probably won't be for a while.
